doc/src/driver.rst

```diff
@@ -61,3 +61,17 @@ Verify each instruction
 .. option::  -r, --reduce
 
 Reduce program and verify
+
+roctx
+----
+
+.. program:: migraphx-driver roctx
+
+| Provides marker information for each operation, allowing MIGraphX to be used with `rocprof <https://rocmdocs.amd.com/en/latest/ROCm_Tools/ROCm-Tools.html>`_ for performance analysis.
+| An example command line combined with rocprof for tracing purposes is given below:
+
+::
+
+    /opt/rocm/bin/rocprof --hip-trace --roctx-trace --flush-rate 1ms --timestamp on -d <OUTPUT_PATH> --obj-tracking on /opt/rocm/bin/migraphx-driver roctx <ONNX_FILE> <MIGRAPHX_OPTIONS>
```

Comment:
Can you give an explanation of what each output file contains? 

After I tried an example with the following command:

```
/opt/rocm/bin/rocprof --hip-trace --roctx-trace --flush-rate 1ms --timestamp on -d . --obj-tracking on ./bin/migraphx-driver roctx ~/Workplace/tests/onnx/alexneti1.onnx
```

I got the following outputs:

```
-rw-rw-r--  1 scxiao scxiao  118784 Nov 15 10:59 results.db
-rw-rw-r--  1 scxiao scxiao    1400 Nov 15 10:59 results.hip_stats.csv
-rw-rw-r--  1 scxiao scxiao  271882 Nov 15 10:59 results.json
-rw-rw-r--  1 scxiao scxiao    2307 Nov 15 10:59 results.stats.csv
-rw-rw-r--  1 scxiao scxiao   17295 Nov 15 10:59 results.sysinfo.txt
```
So I am wondering whether you can explain what are in each file? Actually, I do not know how to open the file "results.db".

Reply:
For the output files, we utilize another repo `rocmProfileData` to get them processed. PR #985 provides all the running and parsing functionalities via `/tools/roctx.py` script, so that we don't need to deal with interpreting the files. I have just rewrote the roctx section in documentation, this should guide user to utilize rocTX as expected.